pkgs/applications/audio/mympd/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, mpd_clientlib
+, openssl
+, lua5_3
+, libid3tag
+, flac
+, mongoose
+}:
+
+stdenv.mkDerivation rec {
+  # Used in ympd's nixos module's systemd's service's ExecStart
+  pname = "mympd";
+  version = "6.5.2";
+
+  src = fetchFromGitHub {
+    owner = "jcorporation";
+    repo = "myMPD";
+    rev = "v${version}";
+    sha256 = "1ykj4lby7yi58kyqiqsk1s5q42diadn3qvmal7bvbvarvlz2088d";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    cmake
+  ];
+  buildInputs = [
+    mpd_clientlib
+    openssl
+    lua5_3
+    libid3tag
+    flac
+  ];
+
+  cmakeFlags = [
+    "-DENABLE_LUA=ON"
+    # Otherwise, it tries to parse $out/etc/mympd.conf on startup.
+    "-DCMAKE_INSTALL_SYSCONFDIR=/etc"
+    # similarly here
+    "-DCMAKE_INSTALL_LOCALSTATEDIR=/var/lib/mympd"
+  ];
+  # See https://github.com/jcorporation/myMPD/issues/315
+  hardeningDisable = [ "strictoverflow" ];
+
+  meta = {
+    homepage = "https://jcorporation.github.io/myMPD/";
+    description = "A standalone and mobile friendly web mpd client with a tiny footprint and advanced features";
+    maintainers = [ stdenv.lib.maintainers.doronbehar ];
+    platforms = stdenv.lib.platforms.unix;
+    license = stdenv.lib.licenses.gpl2;
```

Comment:
`gpl2` is deprecated, see https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206.

Reply:
Thanks :).